### ISSUE TITLE
Why normal deltas have gradients disabled?

### ISSUE THREAD

**Issue by ndming** [Asparagus15/GaussianShader#33]
Hi, thanks for your excellent work. There's a small detail I find hard to understand. In the `training_setup` function, delta normals (`self.normal` and `self.normal2`) have their gradients disabled in `brdf` mode. To my understanding, they should be learnable parameters. Could you help explain this? 

```python
def training_setup(self, training_args):
    # original params setup...

    if self.brdf:
        self._normal.requires_grad_(requires_grad=False) # grad is disabled here
        l.extend([
            {'params': list(self.brdf_mlp.parameters()), 'lr': training_args.brdf_mlp_lr_init, "name": "brdf_mlp"},
            {'params': [self._roughness], 'lr': training_args.roughness_lr, "name": "roughness"},
            {'params': [self._specular], 'lr': training_args.specular_lr, "name": "specular"},
            {'params': [self._normal], 'lr': training_args.normal_lr, "name": "normal"},
        ])
        self._normal2.requires_grad_(requires_grad=False) # as well as here, why?
        l.extend([
            {'params': [self._normal2], 'lr': training_args.normal_lr, "name": "normal2"},
        ])

    # optimizer setup...
```
